frontend/pages/admin/index.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-tabs left color="cyan" dark icons-and-text>
+    <v-tabs-slider color="yellow"></v-tabs-slider>
+
+    <v-tab v-for="tab in tabs" :href="`#tab-${tab.index}`" :key="tab.index">
+      {{ tab.name }}
+      <v-icon>{{tab.icon}}</v-icon>
+    </v-tab>
+
+    <v-tab-item v-for="i in tabs.length" :key="i" :value="'tab-' + i">
+      <v-card flat>
+        <v-card-text v-if="i==1">
+          <Allowance />
+        </v-card-text>
+        <v-card-text v-else-if="i==2">
+          <DistributionConfigure />
+        </v-card-text>
+      </v-card>
+    </v-tab-item>
+  </v-tabs>
+</template>
+
+<script>
+import Allowance from "~/components/admin/allowance.vue";
+import DistributionConfigure from "~/components/admin/distributionConfig.vue";
```

Comment:
Be careful with the name!
```suggestion
import DistributionConfig from "~/components/admin/distributionConfig.vue";
```

Reply:
Resolved. Thanks.